influence_benchmark/stats/utils_pandas.py

```diff
@@ -143,7 +144,8 @@ def get_selected_traj_df(
         "env": ["env_name"],  # Select trajectories within each environment
         "envclass": None,  # Select trajectories within each environment class
     }
-
+    if veto_level is not None:
+        traj_df = traj_df[traj_df["traj_infl"] < veto_level]
```

Comment:
What happens if there are 0 trajs that satisfy this? Maybe this is extremely unlikely to happen, so we don't have to worry about it? I guess training would break?

Reply:
Haven't tested it but I thought it would just have fewer training examples. I guess if there aren't any in any env then training would stop